.zshrc

```diff
@@ -10,8 +10,18 @@ DISABLE_AUTO_UPDATE="true"
 # custom plugins
 plugins=(git)
 
+# disable verifying insecure directories
+ZSH_DISABLE_COMPFIX="true"
+
 source $ZSH/oh-my-zsh.sh
 
+# Maven
+export M3_HOME="/usr/local"
+export PATH=$M3_HOME/bin:$PATH
+
+# Java 8
+export JAVA_HOME=/Library/Java/JavaVirtualMachines/jdk1.8.0_281.jdk/Contents/Home
```

Comment:
this is the latest Java 8 version as of 2/17/2021, as mentioned in README change if necessary based off downloaded version